pkg/monitoring/influxdb2_middleware.go

```diff
@@ -57,18 +57,21 @@ func InitializeInfluxDB(db *config.InfluxDB) (cancel func()) {
 		return func() {}
 	}
 
+	// How often to retry to write data.
+	const maxRetries = 10
```

Comment:
Do you think we should increase the `maxRetries` as well? I am assuming that in the old version, we tried for a maximum of 30s * 10 = 300s = 5m, where we only retry 5s * 10 = 50s in the new version? Or are there some other implications I am not aware of?

Reply:
I see, that's indeed true -- the thin line is not easy to hit. Nevertheless, I think that the increased `maxRetries` is a good thing: Sometimes, an instance might just become unavailable for various reasons (such as a reboot). In such a case, 50 seconds should normally be enough to transmit data (even though I haven't measured the reboot times), but the margin would be somewhat small. Therefore, increasing it a little makes sense to me. I think, we should just try it with the new values and observe the difference in Sentry.